### PR TITLE
Compatibility cleanup

### PR DESCRIPTION
**Pull request by mikaelgoldmann**
- Locate the code handling python 2 / 3 differences in one place.
- Avoid patching imported modules.

### CODE REVIEW

File: submit.py [hunk 1]
```diff
@@ -152,7 +158,7 @@ def get_url(cfg, option, default):
 def get_config():
     """Returns a ConfigParser object for the .kattisrc file(s)
     """
-    cfg = configparser.ConfigParser()
+    cfg = ConfigParser()
```
Comment (austrin) [inline submit.py]: I don't think you tested this.

Comment (mikaelgoldmann) [inline submit.py]: Oh.. Sorry. I changed my mind at one point. Fixed and did manage to submit something.

Comment (austrin) [inline submit.py]: I think the other version, with `import ConfigParser as configparser`, is nicer.

Comment (mikaelgoldmann) [inline submit.py]: OK, can do that. Are you OK with the rest?

Comment (austrin) [inline submit.py]: On 2016-01-26 22:16, Mikael Goldmann wrote:

> OK, can do that. Are you OK with the rest?

Yes, looks fine, assuming you've tested (in both Python 2 and 3).

**Comment by mikaelgoldmann** [comment]
@austrin configparser import as requested. Tested with python 2 and 3.

**Pull request merged**